desktop/datacap/datacap.go

```diff
@@ -92,6 +99,27 @@ func (dc *dataCap) processQuota(out chan<- interface{}, quota *bandwidth.Quota)
 		})
 	}
 }
+func getBandwidth(quota *bandwidth.Quota) (int, int, int) {
```

Comment:
@atavism I think should use what we are using on other platforms just for consistency and we can avoid some platform conditions on the flutter end.